ModTek/Logging/BetterLogSettings.cs

```diff
@@ -0,0 +1,21 @@
+using HBS.Logging;
+
+namespace ModTek.Logging
+{
+    public class BetterLogSettings
+    {
+        public bool Enabled = false;
+        public LogLevel Level = LogLevel.Debug;
+        public string[] IgnoreMessagePatterns = new string[0];
```

Comment:
the settings for message patterns ignore is only applied for the cleaned log

Reply:
Considering we already have stacktrace settings, this settings should be moved at the same location